src/resolvers/line_items.ts

```diff
@@ -13,12 +13,23 @@ import {
   LineItemStatus,
   LineItemUpdateInput,
   MutationResolvers,
+  QueryResolvers,
 } from '../server-internal-types'
 import getPalletWithParentAssociations from './getPalletWithParentAssociations'
 import { authorizeOfferMutation } from './offer_authorization'
 import validateEnumMembership from './validateEnumMembership'
 import validateUris from './validateUris'
 
+const lineItem: QueryResolvers['lineItem'] = async (_, { id }, context) => {
```

Comment:
suggestion (blocking): please add a test case for at least the successful case for this endpoint